src/components/resortPage/SpecificResortPage.js

```diff
@@ -23,7 +23,11 @@ class SpecificResortPage extends Component {
         if (response.ok) {
           return response.json();
         }
-        throw new Error('Something went wrong ...');
+        let message = 'Something went wrong ...';
```

Comment:
- This should be done only in resourceFetcher
- Use Resource fetcher here
- In state set only resort and isLoading